docs/api-guide/testing.md

```diff
@@ -324,6 +324,32 @@ You can use any of REST framework's test case classes as you would for the regul
 
 ---
 
+# URLPatternsTestCase
+
+REST framework also provides a test case class for isolating `urlpatterns` on a per-class basis. Note that this inherits from Django's `SimpleTestCase`, and will most likely need to be mixed with another test case class.
+
+## Example
+
+    from django.urls import include, path, reverse
+    from rest_framework.test import APITestCase, URLPatternsTestCase
+
+
+    class AccountTests(APITestCase, URLPatternsTestCase):
+        urlpatterns = [
+            path('api/', include('api.urls')),
+        ]
+
+        def test_create_account(self):
+            """
+            Ensure we can create a new account object.
+            """
+            url = reverse('account-list')
+            response = self.client.get(url, format='json')
+            self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
how about using pytest style asserts?

Reply:
Consistency with the other test case docs. That said, the testing doc is inconsistent in its use of assertions, using a mix of both pytest and unittest assertions.